src/components/PeopleList.tsx

```diff
@@ -0,0 +1,88 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  listOfPeople: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ listOfPeople }) => {
+  const { selectedSlug = 0 } = useParams();
+
+  const list = listOfPeople.map(personAge => {
+    const mother = listOfPeople.find(woman => woman.name
+      === personAge.motherName);
+    const father = listOfPeople.find(man => man.name
+      === personAge.fatherName);
+
+    return {
+      ...personAge,
+      mother,
+      father,
+    };
+  });
```

Comment:
Move it to separate util (flie), use destruction like this
  `const list = listOfPeople.map(({motherName, fatherName})=>`

```
const findParent = (list: Person[], person: Person , parent: 'motherName' | 'fatherName' ) => (
  list.find(({name}) => name === person[parent] )
)
```